docker/admin-dev/Dockerfile.mako

```diff
@@ -0,0 +1,6 @@
+FROM camptocamp/geomapfish-commons:${major_version}
```

Comment:
Note that we already have a dockerfile for admin dev in admin folder

Reply:
Where is it used ? I do not found

Reply:
This one don't has the same goal, this one is a build container, in the admin interface it's the one to run the admin in dev mode.

Reply:
Could it be named admin-build in that case, not admin-dev

Reply:
ok :-)